katalog/etcd-backup/etcd-backup-s3/rclone/rclone.conf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+[minio]
+type = s3
+provider = Minio
+
+# Those are real Minio credentials to Minio's Playground. You can use them to test the backup.
+# https://min.io/docs/minio/linux/administration/minio-console.html#logging-in
+access_key_id = Q3AM3UQ867SPQQA43P2F
+secret_access_key = zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
+
+endpoint = https://play.min.io:9000
```

Comment:
I don't like having real credentials here. It's better to use placeholders and to test the backup on the e2e tests, deploying an internal Minio bucket.

We should also add e2e tests if possible, IF with kind we can do something like that.

Reply:
Those are public credentials to a public instance of Minio, they use it for example testing, but I get your point. The link reported there points to the very same credentials, so we might drop the hardcoded credentials and add a comment suggesting that if users want to test the feature, they can find a public Minio test server with public credentials inside the link.